src/components/Layout/HomeContent.js

```diff
@@ -1612,13 +1606,13 @@ function Thumbnail({video}) {
                 key={i}
                 className="h-8 w-8 border-2 shadow-md border-gray-70 object-cover rounded-full"
                 src={src}
-                alt=""
+                alt="Speaker"
```

Comment:
I'd say this be better left empty or perhaps actually add an alt description to our `videos` data list, or perhaps more generically "video thumbnail"?

Reply:
This too has been deleted